Automattic/newspack-plugin
# feat: campaign group filter

### All Submissions:






### Changes proposed in this Pull Request:

Adds UI to filter campaigns by campaign group taxonomy.

Dependent PRs: 
~https://github.com/Automattic/newspack-plugin/pull/764~
~https://github.com/Automattic/newspack-plugin/pull/767~
~https://github.com/Automattic/newspack-plugin/pull/768~

<img width="1301" alt="Screen Shot 2021-01-04 at 8 30 56 AM" src="https://user-images.githubusercontent.com/1477002/103540114-356f0b80-4e67-11eb-97e4-c5dfcef5862b.png">

### How to test the changes in this Pull Request:

1. Make sure there are several Campaign Group taxonomy terms created, which can be done by editing any campaign and using the Campaign Groups sidebar panel.
2. In the Campaigns Wizard, observe `Campaign groups` `SelectControl`. Verify it contains all campaign groups. 
3. Change the selected campaign group, verify that visible campaigns are filtered correctly.
4. Change back to `All Campaigns`, verify all campaigns are shown again.

### Other information:
